sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1315,6 +1315,12 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
     return getInfo(param_value_size, param_value, param_value_size_ret,
                    capabilities);
   }
+  case PI_EXT_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES:
+  case PI_EXT_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES:
+    // There is no way to query this in the backend
+    setErrorMessage("This backend does not support this query",
```

Comment:
nit: I would mention the backend explicitly for a clean message sake, something along the lines of "PI HIP backend does not support this query".
Same for HIP backend.